### ISSUE TITLE
Size issue

### ISSUE THREAD

**Issue by Dhruvkanani** [Chrisvin/FlipTabs#2]
For example, If i add a fliptab of width 100dp and height 60dp, then also the resulted fliptab is of approximately height 50% of given height and width is approximately 20dp less than the original width given

**Comment by Chrisvin** [comment]
`FlipTab` has a default padding of 16dp set on all sides to make room for the flipping animation.
In retrospect, I should have probably mentioned that somewhere in the readme and added an option to disable it.

I've added a boolean attribute `removeDefaultPadding` that can be set to remove the default padding.
```
<com.jem.fliptabs.FlipTab
           ...
            app:removeDefaultPadding="true"
            ... />
```

**Note**: Disabling the default padding means that you'll need to ensure that there is enough space around the `FlipTab` so as to not clip the flipping animation.

**Comment by Chrisvin** [comment]
This has been fixed in release `v1.2`.
